src/main/webapp/startup.js

```diff
@@ -0,0 +1,35 @@
+import Renderer from './Renderer.js';
+
+// Events array for testing
+var events = { "list": [
+    {
+        "name": "Breakfast",
+        "location": "Zurich",
+        "time": "9:30AM - 10:30AM"
+    },
+    {
+        "name": "Lunch",
+        "location": "Stuttgart",
+        "time": "1:30PM - 2:30PM"
+    },
+    {
+        "name": "Dinner",
+        "location": "Bucharest",
+        "time": "7:30PM - 8:30PM"
+    }]
+}
```

Comment:
Suggestion: maybe you should opt out of using the outer object and instead of:
`
events = {
      list: [...]
};
`

You do it like:
`events = [...];`